src/resources/Transfers/types.ts

```diff
@@ -0,0 +1,150 @@
+import { ISODateString } from '../../utils/types'
+
+type TransferType = 'URL' | 'TORRENT' | 'PLAYLIST' | 'LIVE_STREAM'
+
+type TransferStatus =
+  | 'WAITING'
+  | 'IN_QUEUE'
+  | 'DOWNLOADING'
+  | 'COMPLETING'
+  | 'STOPPING'
+  | 'SEEDING'
+  | 'COMPLETED'
+  | 'ERROR'
+  | 'PREPARING_SEED'
+
+type BaseTransfer = {
+  availability: null | number
+  created_at: ISODateString
+  current_ratio: null | number
+  down_speed: number
+  downloaded: number
+  error_message: null | string
+  estimated_time: null | number
+  file_id: null | number
+  finished_at: null | ISODateString
+  id: number
+  is_private: boolean
+  name: string
+  save_parent_id: number
+  seconds_seeding: null | number
+  size: number
+  source: string
+  started_at: null | ISODateString
```

Comment:
when does it become a `string` from null? if it's `IN_QUEUE` or `DOWNLOADING`?

Reply:
`IN_QUEUE` iken bir deger set ediliyor aslinda. Fakat henuz download baslamadan `ERROR` olarak isaretlendiyse (mesela kullanicinin plani yoksa) bu durumda set edilmiyor.

Bu deger ne zaman set edildigi client icin onemli bir bilgi mi?